cypress/integration/podcasts.spec.js

```diff
@@ -156,6 +156,25 @@ describe("newsdesk logged in", () => {
     cy.get("section[slotno=7] button[title='Delete']").click()
     cy.get("section[slotno=7]").should("not.contain", textToCopy)
   })
+
+  it("should dragging item in inbox", () => {
+    switchPodCast("dev sandbox")
+    cy.get("section[slotno=7] textarea").type("dragging item in inbox{enter}", {
+      force: true,
+    })
+    cy.get("section[slotno=7] button[title='Share to podcast']").click()
+    cy.get("section[slotno=7] input[id='dev'][type='checkbox']").click()
+    cy.get("section[slotno=7] textarea").type("dragging item{enter}", {
+      force: true,
+    })
+    cy.get("section[slotno=7]").should("contain", "dragging item")
+    cy.get("section[slotno=7] button[title='Share to podcast']").eq(0).click()
+    cy.get("section[slotno=7] input[id='dev'][type='checkbox']").click()
+    cy.get("#inbox-column").should("contain", "dragging item")
+    cy.get("#inbox-column ul li span div")
+      .eq(0)
+      .trigger("dragend", { clientX: 83, clientY: 153 }, { force: true })
```

Comment:
Needs to test that it actually dragged

Strongly recommend swapping the x and y to location of an element you get with a lookup otherwise this and other tests will be fragile

Reply:
I have checked this test and is working well.

Reply:
have opened #295 to address this and others